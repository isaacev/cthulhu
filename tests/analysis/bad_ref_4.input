use ::io;

fn main() -> () {
  io::println::extra();
}
