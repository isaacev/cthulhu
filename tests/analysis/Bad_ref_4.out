
  UNKNOWN FIELD  -------------------------------------------------------------

  Reference to an unknown field in a namespace.

    : Bad_ref_4
  5 |   Fmt::Bool(true);
    |        ^^^^

